src/data/parsing.py

```diff
@@ -85,80 +95,117 @@ def create_parse_files(doc, docNum, writeFile = True, pathToFolders=''):
                         t3=list(t3)
 
                         #current position is the threshold for change
-                        if (t2[1] - lastAdjustedPosition) > int(token.index) and t1 not in ['combine', 'compare', 'function', 'transform']:
+                        if (t2[1]-lastAdjustedPosition)>int(token.index) and

```

Comment:
On line 98 and 101 we need a '\' after the 'and' or 'and' needs to start the next line, otherwise will result in a syntax error.

Reply:
good catch; wrapped the entire `if` statement in parentheses, which will allow for the line break (and PEP-8 compliance!).